codecov.yml

```diff
@@ -1,4 +1,5 @@
 codecov:
+  token: 7d2ecd8b-ac89-4c2d-ad05-119518130cb2
```

Comment:
visible token is required for fork workflow. I dont think we care that much if other starts pushing to our codecov.